webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -957,6 +957,17 @@ msgstr ""
 "Der Steuerlotse bietet Ihnen dafür allerdings derzeit keine "
 "Möglichkeiten."
 
+#: app/forms/steps/eligibility_steps.py:748
+msgid "form.eligibility.result-note.capital_investment"
+msgstr ""
+"Die Besteuerung von Kapitalerträgen erledigen Banken und Versicherungen "
+"für Sie. Wenn Sie Freistellungsaufträge erteilt haben, dann erhalten Sie "
+"dort auch Ihren Sparerfreibetrag. Wenn Sie keine Freistellungsaufträge "
+"erteilt haben oder mit der Höhe der für Sie abgeführten Steuern nicht "
+"einverstanden sind, dann können Sie in der Einkommensteuererklärung eine "
+"Änderung der Steuerbelastung beantragen. Der Steuerlotse bietet Ihnen "
+"dafür allerdings derzeit keine Möglichkeiten."
```

Comment:
Why is the cheaper_check message not gone? Is it used anywhere else or did you just not commit the changes to this file?

Reply:
Good catch. I did not commit the changes